test/bundle.test.ts

```diff
@@ -55,33 +55,12 @@ describe.skipIf(process.env.SKIP_BUNDLE_SIZE === 'true' || process.env.ECOSYSTEM
         "@vue/runtime-dom",
         "@vue/server-renderer",
         "@vue/shared",
-        "cookie-es",
-        "debug",
-        "defu",
-        "destr",
         "devalue",
         "estree-walker",
-        "h3",
-        "has-flag",
         "hookable",
-        "http-graceful-shutdown",
-        "iron-webcrypto",
-        "klona",
-        "ms",
-        "node-fetch-native",
-        "ofetch",
-        "ohash",
-        "pathe",
-        "radix3",
-        "scule",
         "source-map-js",
-        "supports-color",
         "ufo",
```

Comment:
interesting we have `hookable` and `ufo` here, though they are listed as runtime dependencies to-be-bundled [here](https://github.com/unjs/nitro/pull/1554/files).

Reply:
Yes, that’s something i have to investigate after upstream changes..

Reply:
BTW it makes sense to happen if we are externaling a lib that depends on them. (unhead maybe?)

Reply:
Post-inlinig is not possible.

I would recommend to inline known nuxt runtime dependencies as well to leverage same benefits of upstream. (or disable this optimization totally up to you!)